operator/helper/parser.go

```diff
@@ -82,6 +81,10 @@ func (c ParserConfig) Build(logger *zap.SugaredLogger) (ParserOperator, error) {
 		parserOperator.TraceParser = c.TraceParser
 	}
 
+	if c.ScopeNameParser != nil {
+		parserOperator.ScopeNameParser = c.ScopeNameParser
+	}
```

Comment:
Unrelated change?

Reply:
Sort of. A bug I found while updating tests.

I'll split it into a separate PR.

Reply:
https://github.com/open-telemetry/opentelemetry-log-collection/pull/465